src/Function/Consequences.agda

```diff
@@ -54,3 +55,24 @@ module _
   contraInjective : Injective _≈₁_ _≈₂_ f →
                     ∀ {x y} → ¬ (x ≈₁ y) → ¬ (f x ≈₂ f y)
   contraInjective inj p = contraposition inj p
+
+∘-cong : ∀ {f} {g} {≈₁ : Rel A ℓ₁} {≈₂ : Rel B ℓ₂} →
+          Congruent ≈₁ ≈₁ f → Congruent ≈₁ ≈₂ g → Congruent ≈₁ ≈₂ (g ∘ f)
+∘-cong f-cong g-cong = g-cong ∘ f-cong
+-- ∘-cong = flip _∘_  -- Yields implicit/explicit function type error.
```

Comment:
That's because `_∘_` is the dependent version and therefore you can't flip it. If you use the primed version "∘\prime" then you should be able to flip it.

Reply:
This lemma already exists though, in `Function.Construct.Composition`, which is where the `∘-cong₂` should live as well if we keep it.